.vimrc

```diff
@@ -0,0 +1,114 @@
+" Use Vim settings, rather then Vi settings. This setting must be as early as
+" possible, as it has side effects.
+set nocompatible
+
+" Leader
+let mapleader = " "
+
+" Switch between the last two files
+nnoremap <leader><leader> <c-^>
+
+" Set undo file to allow undoing even after closing and reopening vim
+silent !mkdir -p ~/.vim/undo > /dev/null 2>&1
+set undofile
+set undodir=~/.vim/undo
+
+" Softtabs, 2 spaces
+set softtabstop=2
+set tabstop=2
+set shiftwidth=2
+set shiftround
+set expandtab
+set backspace=2   " Backspace deletes like most programs in insert mode
+
+" Numbers
+set relativenumber
+set numberwidth=5
+
```

Comment:
Occasionally, I'll open up a log file with vim if I feel like I'm going to have to go through a lot of history, but I can get by with `less` in those situations just as well (if not better).